src/wp-admin/nav-menus.php

```diff
@@ -580,10 +580,12 @@
 	'under'                   => __( 'Under %s' ),
 	/* translators: %s: Previous item name. */
 	'outFrom'                 => __( 'Out from under %s' ),
-	/* translators: 1: Item name, 2: Item position, 3: Total number of items. */
-	'menuFocus'               => __( '%1$s. Menu item %2$d of %3$d.' ),
-	/* translators: 1: Item name, 2: Item position, 3: Parent item name. */
-	'subMenuFocus'            => __( '%1$s. Sub item number %2$d under %3$s.' ),
+	/* translators: 1: Item name, 2: Item type, 3: Item index, 4: Total Items */
+	'menuFocus'               => __( 'Edit %1$s (%2$s, %3$d of %4$d)' ),
```

Comment:
In `4: Total Item` the word Item should start with lowercase and end with a period.